test/fixtures/articles.yml

```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  title: MyString
+  body: MyText
+
+two:
+  title: MyString
+  body: MyText
```

Comment:
You can delete test folder because you will be cover by tests your project with rspec gem

Reply:
Fixed